### DISCUSSION TITLE
Support 2FA?

### DISCUSSION THREAD

**Discussion by russdot** [andrewdwallo/filament-companies#98] [Feature requests]
I was curious about the removal of Fortify and 2FA support on 3.x? Any plans for 2FA support?

**Comment by andrewdwallo** [comment]
Fortify was removed because Filament already supports all of Fortify's features except for 2FA. Keeping Fortify would only add bloat to the package in this case. If you want 2FA support, then PR's are more than welcome!

**Reply by russdot** [reply]
Fair enough! Appreciate the quick reply.
